echo/sentryecho.go

```diff
@@ -86,3 +111,12 @@ func GetHubFromContext(ctx echo.Context) *sentry.Hub {
 	}
 	return nil
 }
+
+// GetTransactionFromContext retrieves attached *sentry.Span instance from echo.Context.
+// If there is no transaction on echo.Context, it will return nil.
+func GetTransactionFromContext(ctx echo.Context) *sentry.Span {
+	if span, ok := ctx.Get(transactionKey).(*sentry.Span); ok {
+		return span
+	}
+	return nil
+}
```

Comment:
We either need to make sure this only returns the root span or we rename it to `GetSpanFromContext`

Reply:
Ah, I'll rename it to `GetSpanFromContext` then. I think that'll make the transition to otel much easier.